vignettes/sd_iv_rich_pkpd.Rmd

```diff
@@ -0,0 +1,90 @@
+---
+title: "Single Dose IV - PKPD"
+author: "Vignette Author"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Single Dose IV - PKPD}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r echo=FALSE}
+library(DiagrammeR)
```

Comment:
we'll need to add DiagrammeR to Suggests so the vignette can be knit. Really like the idea